core/iwasm/common/wasm_native.c

```diff
@@ -425,6 +428,13 @@ wasm_native_init()
         goto fail;
 #endif /* WASM_ENABLE_LIB_RATS */
 
+#if WASM_ENABLE_WASI_NN != 0
+    n_native_symbols = get_wasi_nn_export_apis(&native_symbols);
+    if (!wasm_native_register_natives("wasi_nn", native_symbols,
```

Comment:
@tonibofarull where did this "wasi_nn" module name come from?
can you give me a reference to an exact version of the spec this was based on?
eg. commit id of [wasi-nn repo](https://github.com/WebAssembly/wasi-nn)
as far as i researched wasi-nn has always been using "wasi_ephemeral_nn".

Reply:
It was an arbitrary name as I couldn't get an API list based on witx/wit.

Reply:
ok. so this is not compatible with any versions of wasi-nn. right?

Reply:
Compatibility is not guaranteed.

Reply:
So, there will be no harm if we replace *wasi-nn* with *wasi_ephemeral_nn* in the repo?

Reply:
> Compatibility is not guaranteed.

well, actually incompatibility is guaranteed because of the different module name, isn't it?